src/App/Settlements/actions.ts

```diff
@@ -10,22 +10,18 @@ import {
   SET_SETTLEMENTS_FILTER_VALUE,
   CLEAR_SETTLEMENTS_FILTERS,
   SELECT_SETTLEMENT,
-  SET_SETTLEMENT_DETAILS,
```

Comment:
https://github.com/mojaloop/reporting-hub-bop-role-ui/blob/master/src/App/Users/slice.ts

Theres this nice `slice` utility fuction that creates actions with the same name as your reducers and don't need to mantain a list of action constants. 

A nice to have, that is not needed in this PR.

Reply:
Ah yeah, great to know about, thank you. Definitely good to reduce the toil of working with redux :grimacing:

Reply:
As you say, I won't make that change in this PR. I'll keep this one focused :joy: . Okay, kidding, but I'll fold that into something else later. Thanks again.